packages/amplify-cli-core/API.md

```diff
@@ -254,7 +254,8 @@ export interface AmplifyInternalOnlyPostEnvRemoveEventData {
 // @public (undocumented)
 export class AmplifyNodePkgDetector {
     // (undocumented)
-    detectAffectedDirectDependencies: (dependencyToSearch: string) => Array<DetectedDependency> | [];
+    detectAffectedDirectDependencies: (dependencyToSearch: string) => Array<DetectedDependency> | [
```

Comment:
It seems that there are couple of API.md changes that were not generated properly and are failing pull request checks.

Please run:
```
yarn clean && yarn && yarn build && yarn extract-api
```
and amend PR to fix this.